README.md

```diff
@@ -271,6 +271,8 @@ Not at the moment. It relies on XCode and associated libraries (SpriteKit, Scene
 
 - Can I use it for commercial projects?
 
-Yes, you can, but please, check the License details (TBD) to know the details.
+Yes, you can, but please, check the [License details](LICENSE) to know the details.
 
- 
\ No newline at end of file
+## 9. License
+
+The Boken Engine framework is available under the [Mozilla Public License Version 2.0](https://mozilla.org/MPL/2.0/).
```

Comment:
Include a link to the `doc/choose-a-license.md` document so a reader can also know the rationale and background of this choice.

Reply:
Good idea! I've added an additional text so now they're linked. Thanks!